events/code_review_events/workflow.py

```diff
@@ -219,7 +225,44 @@ def publish_results(self, payload):
         return True
 
     def parse_pulse(self, payload):
-        assert self.publish is True, "Publication disabled"
+
+        routing = payload["routing"]
+
+        # Process autoland payloads
+        if routing["exchange"] == PULSE_TASK_GROUP_RESOLVED:
```

Comment:
It'd be nicer to have this directly in libmozevent, so we don't have to implement the routing ourselves here. Can you file a follow-up?

Reply:
https://github.com/mozilla/libmozevent/issues/28